src/utils.ts

```diff
@@ -13,3 +15,40 @@ export const aestheticBaseAttributes: Pick<Nft, 'aestheticRank' | 'aestheticRank
     timesWon: 0,
     aestheticRankReliability: 0,
 };
+
+/**
+ * A mapping of all PopRank supported chains to all
+ * of our supported marketplaces.
+ *
+ * The first marketplace under a chain is considered the "default".
+ */
+export const chainMarketplaceMap: ChainMarketplaceMap = {
+    'ethereum': {
+        'opensea': {
+            name: "OpenSea",
+            nftUrlFormatter: (address?: string, id?: string) => `${OPENSEA_BASE_NFTS_URL}/${address}/${id}`,
+            collectionUrlFormatter: (address: string) => `${OPENSEA_BASE_COLLECTIONS_URL}/${address}`
+        }
+    },
+    'polygon': {
+        'opensea': {
+            name: "OpenSea",
+            nftUrlFormatter: (address?: string, id?: string) => `${OPENSEA_BASE_NFTS_URL}/matic/${address}/${id}`,
+            collectionUrlFormatter: (address: string) => `${OPENSEA_BASE_COLLECTIONS_URL}/${address}`
+        }
+    },
+    'solana': {
+        'magiceden': {
```

Comment:
when we list a collection like DeGods which is present on both OpenSea and MagicEden, we'd like to show links to both marketplaces -- can we also include an 'opensea' entry here?

Reply:
you got it king